pkgs/applications/video/kodi-packages/vfs-sftp/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, rel, buildKodiBinaryAddon, fetchFromGitHub, openssl, libssh, zlib }:
+buildKodiBinaryAddon rec {
+  pname = namespace;
+  namespace = "vfs.sftp";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "xbmc";
+    repo = namespace;
+    rev = "${version}-${rel}";
+    sha256 = "06w74sh8yagrrp7a7rjaz3xrh1j3wdqald9c4b72c33gpk5997dk";
+  };
+
+  meta = with lib; {
+    description = "SFTP Virtual Filesystem add-on for Kodi";
+    license = licenses.gpl2Plus;
+    platforms = platforms.all;
```

Comment:
The platform should be probably taken from kodi instead.

Reply:
In theory it should not. Binary plugins are available for specific systems. In practice we only ship `kodi` for `linux` so that would work... but I think it is better flexibility to keep it open and specified per binary plugin. Will leave that as is.